imports/model-helpers.ts

```diff
@@ -27,7 +27,7 @@ const huntsMatchingCurrentUser = function <T extends HuntModel> (
   // As a note: this will not re-publish if the user's hunt membership
   // changes, so use it carefully (basically, use it when you already
   // know the user is a member of the hunt in question).
-  const u = MeteorUsers.findOne(this.userId);
+  const u = MeteorUsers.findOne(this.userId!);
```

Comment:
It's not clear to me that `this.userId` is guaranteed to be a string here.  In fact, we appear to be checking that we got a user, and I think this would just turn into `MeteorUser.findOne(undefined)`, which might just return an arbitrary user?

Either way I'd be happier if we just checked `this.userId` explicitly in this function instead of writing `this.userId!`